src/Uno.UI/Uno.UI.csproj

```diff
@@ -32,6 +32,7 @@
 
 	<ItemGroup>
 		<Compile Remove="Mixins\**\*.cs" />
+		<None Remove="UI\OnSystemUiVisibilityChangeListener.Android.cs" />
```

Comment:
don't modify this file

Reply:
Happens every time I add a file. Is there any way to avoid that ?

Reply:
Not as far as I know, it seems to be a quirk of Visual Studio when confronted with Uno's wildcard-based scheme. Just make sure not to include changes to the csproj file in your commits.